pgrx-tests/src/framework.rs

```diff
@@ -420,26 +482,40 @@ fn initdb(postgresql_conf: Vec<&'static str>) -> eyre::Result<()> {
 }
 
 fn modify_postgresql_conf(pgdata: PathBuf, postgresql_conf: Vec<&'static str>) -> eyre::Result<()> {
-    let mut postgresql_conf_file = std::fs::OpenOptions::new()
-        .write(true)
-        .truncate(true)
-        .open(format!("{}/postgresql.auto.conf", pgdata.display()))
-        .wrap_err("couldn't open postgresql.auto.conf")?;
-    postgresql_conf_file
-        .write_all("log_line_prefix='[%m] [%p] [%c]: '\n".as_bytes())
-        .wrap_err("couldn't append log_line_prefix")?;
+    let mut contents = String::new();
 
+    contents.push_str("log_line_prefix='[%m] [%p] [%c]: '\n");
+    contents
+        .push_str(&format!("unix_socket_directories = '{}'\n", pgdata.parent().unwrap().display()));
```

Comment:
I probably should have asked earlier, but why are we adding this `unix_socket_directories` field? We did not do this before.

Reply:
wait, yes we did. no, the problem is now it's different somehow...

Reply:
Sorry for the noise, figured out what was up here: adding `"pg"` to the string used up someone's last two bytes. :smiling_face_with_tear: